Sources/Plasma/PubUtilLib/plAvatar/plAvatarClothing.cpp

```diff
@@ -910,15 +910,13 @@ void plClothingOutfit::WriteToVault(const TArray<plStateDataRecord*> & SDRs)
     // Get all existing clothing SDRs
     rvn->GetChildNodes(plVault::kNodeType_SDL, 1, &nodes);    // REF: Find
 
-    const TArray<plStateDataRecord*> * arrs[] = {
+    const std::vector<plStateDataRecord*> * arrs[] = {
         &SDRs,
         &morphs,
     };
-    for (unsigned arrIdx = 0; arrIdx < std::size(arrs); ++arrIdx) {
-        const TArray<plStateDataRecord*> * arr = arrs[arrIdx];
-        
+    for (const auto * arr : arrs) {
         // Write all SDL to to the outfit folder, reusing existing nodes and creating new ones as necessary
-        for (unsigned i = 0; i < arr->Count(); ++i) {
+        for (unsigned i = 0; i < arr->size(); ++i) {
```

Comment:
Might as well go ahead and turn this into a range-based for. @dpogue will be happy...

Reply:
Hmm, I thought I converted all the simple cases, but I guess I missed this one.